Cannot find module '@testing-library/jest-dom/matchers'

I am getting the error 

Cannot find module '@testing-library/jest-dom/matchers' from 'selector.test.js'

are the docs correct it seems to work when I do 
import {
  toBeInTheDocument
} from '@testing-library/jest-dom

<img width="311" alt="image" src="https://user-images.githubusercontent.com/16217396/73485333-c1bfa700-4357-11ea-9376-7c8c13c36356.png">


<img width="216" alt="image" src="https://user-images.githubusercontent.com/16217396/73485423-e6b41a00-4357-11ea-9edd-c37b6f5ec4fb.png">

--- a/matchers.js
+++ b/matchers.js
@@ -1,1 +1,2 @@
-export * from './src/matchers'
+const matchers = require('./dist/matchers')
+module.exports = matchers

--- a/package.json
+++ b/package.json
@@ -24,7 +24,8 @@
   },
   "files": [
     "dist",
-    "extend-expect.js"
+    "extend-expect.js",
+    "matchers.js"
   ],
   "keywords": [
     "testing",

